Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -1269,6 +1269,18 @@
       - ClothingOuterWinterCentcom
       - ClothingOuterWinterSyndie
       - ClothingOuterWinterSyndieCap
+      - ClothingUniformInterdyneChemist #Delta-V Interdyne Chemist Jumpsuit
+      - UniformScrubsColorCybersun #Delta-V Cybersun Surgeon Jumpsuit
+      - ClothingUniformCybersunAttorney #Delta-V Cybersun Lawyer Jumpsuit
+      - ClothingHeadHatSurgcapCybersun #Delta-V Cybersun Surgeon Cap
+      - ClothingOuterInterdyneChemistrySuit #Delta-V Interdyne Outerwear
+      - ClothingOuterCoatCybersunWindbreaker #Delta-V Cybersun Surgeon Outerwear
+      - ClothingMaskInterdyneChemistry #Delta-V Interdyne Gasmask
+      - ClothingHeadHatSurgcapCybersun #Delta-V Cybersun Surgeon Cap
+      #- ClothingUniformCybersunHazard #Delta-V Cybersun Engineering Jumpsuit
+      #- ClothingUniformCybersunCasual #Delta-V Cybersun Casual Jumpsuit
+      #- ClothingUniformCybersunRND #Delta-V Cybersun Researcher Jumpsuit
+      #- ClothingOuterCybersunOvercoat #Delta-V Cybersun Lawyer Overcoat
```

Comment:
just do  a Begin DeltaV Additions and End DeltaV Additions comment at the start and end, not on every individual line

Reply:
this is how i saw it done with the CJ outfits, Ill do that instead